app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    if command_list[0] == "mv":
+        new_path = command_list[2].split("/")
+        dir_file = ""
+        for i in range(len(new_path) - 1):
+            dir_file = dir_file + new_path[i]
+            if not os.path.exists(dir_file):
+                os.mkdir(dir_file)
```

Comment:
Use os.mkdirs with argument `exist_ok=True` to avoid exception if directory exists